File: backend/Modulos/models/traspasos.go
package models

// Traspasos is...
type Traspasos struct {
	IDTraspaso      int    `json:"idtraspaso"`
	ServicioOrigen  int    `json:"servicioorigen"`
	AbrevSrvOrigen  string `json:"abrevsrvorigen"`
	DescSrvOrigen   string `json:"descsrvorigen"`
	ServicioDestino int    `json:"serviciodestino"`
	AbrevSrvDestino string `json:"abrevsvrdestino"`
	DescSrvDestino  string `json:"descsrvdestino"`
	FechaTraspaso   string `json:"fechatraspaso"`
	ResponTraspaso  string `json:"respontraspaso"`
	ObservTraspaso  string `json:"observtraspaso"`
}

File: frontend/src/app/models/entity/Traspasos.ts
export class Traspasos{
    constructor (
        public idtraspaso       ?: number, 
        public servicioorigen   ?: number,
        public abrevsrvorigen   ?: string,
        public descsvrorigen    ?: string,
        public serviciodestino  ?: number,
        public abrevsrvdestino  ?: string,
        public descsvrdestino   ?: string,
        public fechatraspaso    ?:string  ,
        public respontraspaso   ?:string,
        public observtraspaso   ?:string,
        //public canttraspaso     ?: string,
        public bodegaorigen     ?: number,
        public bodegadestino    ?: number
    ) {}
}